alacritty.yml

```diff
@@ -291,6 +291,8 @@ mouse:
     # This program is executed when clicking on a text which is recognized as a URL.
     # The URL is always added to the command as the last parameter.
     #
+    # When set to `None`, URL launching will be disabled completely.
+    #
```

Comment:
Would it be possible to set the value `None` on the whole `url` key? That might be even better.

Reply:
It would be possible, however that might cause issues in the future I'd imagine.

Like this, it disables one specific feature, setting the whole `url` key to `None` would potentially disable multiple features.

Imagine we one day introduce a `url.copy_to_clipboard` option, which just copies the URL to the clipboard when clicking on it. Then it wouldn't be possible to disable just one of these.

So I think the current way is a good way to go about it.